pkg/entities/entity_integration_test.go

```diff
@@ -172,7 +177,10 @@ func TestIntegrationGetEntity_ApmEntity(t *testing.T) {
 	client := newIntegrationTestClient(t)
 
 	// GUID of Dummy App
-	result, err := client.GetEntity(testhelpers.IntegrationTestApplicationEntityGUID)
+	// result, err := client.GetEntity(testhelpers.IntegrationTestApplicationEntityGUID)
+
+	// Temporarily using the GUID of a different app
+	result, err := client.GetEntity("MzgwNjUyNnxBUE18QVBQTElDQVRJT058NTUzNDQ4MjAy")
```

Comment:
`func TestIntegrationGetEntity_ApmEntity(t *testing.T)`
- _temporarily_ changed the GUID of the entity being used by the test to fetch details of, which is a Python-based APM entity, `Dummy App Pro Max`
- accordingly, changed assert statements in the following lines of the same function to match the ID of the entity, language, etc.